settings.py

```diff
@@ -185,6 +185,7 @@
 }
 
 TERRA_API_ROOT_URL = os.environ.get('TERRA_API_ROOT_URL')
+ANVIL_BASE_URL = 'https://anvil.terra.bio/'
```

Comment:
you should update communication_utils.py to use this constant instead of having it defined separately there